lib/scarpe/glibui/document_root.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class Scarpe
+  class GlimmerLibUIDocumentRoot < Scarpe::GlimmerLibUIWidget
+    include Scarpe::GlimmerLibUIBackground
+
+    attr_reader :debug
+    attr_reader :redraw_requested
+
+    def initialize(properties)
+      @callbacks = {}
+
+      super
+    end
+
+    def display
+      <<~GTEXT
+      window {
+        area {
+          #{@children.map(&:display).join}
+        }
+      }.show
+      GTEXT
+    end
+
+    def element(&blck)
+      window(&blck).show
+    end
+
+    # Bind a Scarpe callback name; see Scarpe::Widget for how the naming is set up
+    def bind(name, &block)
```

Comment:
Not sure if bind needs to leave altogether or if this one is wanted generally

cc @noahgibbs

Reply:
If you're going to convert Scarpe handlers into Glimmer handlers and handle them 100% in Glimmer then Scarpe doesn't need bind. I think that may be easier said than done, though.

Reply:
Thanks for the context. Let's keep this in for now. Any changes can be separate issues.